applications/ParticleMechanicsApplication/tests/test_generate_mpm_particle_condition.py

```diff
@@ -31,6 +31,10 @@ def _generate_particle_condition_and_check(self, current_model, dimension, geome
         for condition in grid_model_part.Conditions:
             condition.SetValue(KratosParticle.PARTICLES_PER_CONDITION, num_particle)
             condition.SetValue(KratosParticle.MPC_BOUNDARY_CONDITION_TYPE, 1)
+            if (geometry_element == "Point"):
+                condition.SetValue(KratosParticle.MPC_IS_NEUMANN, True)
+                condition.SetValue(KratosParticle.POINT_LOAD, [1.0,0,0])
```

Comment:
I'm not sure if 1.0 is a good value or if you want no make it 0.0. As far as I remember there are processes that add up values instead of replacing them.

Reply:
Yes I'll do that - it has no effect anyway :)